pkg/internal/apis/config/validate.go

```diff
@@ -49,13 +50,15 @@ func (c *Cluster) Validate() error {
 		}
 	}
 
+	isDualStack := c.Networking.IPFamily == DualStackFamily
```

Comment:
should we validate against the kubernetes version?

Reply:
config doesn't know the kubernetes version, so no.

we can check later, when we get nodes.

Reply:
kubeadm will fail then

Reply:
we can still check prior to kubeadm, but after creating node containers. this can be in a follow-up.